tests/TestCase/ORM/Behavior/TreeBehaviorTest.php

```diff
@@ -1175,6 +1176,32 @@ public function testDeleteSubTreeScopedTree(): void
         $this->assertMpttValues($expected, $table);
     }
 
+    /**
+     * Tests deleting a subtree with ORM delete operations
+     */
+    public function testDeleteSubTreeWithOrm(): void
+    {
+        $NumberTreesArticles = $this->getTableLocator()->get('NumberTreesArticles');
+        $table = $this->table;
+        $table->addAssociations([
+            'hasMany' => [
+                'NumberTreesArticles' => [
+                    'cascadeCallbacks' => true,
+                    'dependent' => true,
+                ],
+            ],
+        ]);
+        $table->getBehavior('Tree')->setConfig(['useOrmDelete' => true]);
+        $entity = $table->get(1);
+        $this->assertTrue($table->delete($entity));
+
+        $expected = [
+            ' 5: 6 - 11:alien hardware',
+        ];
+        $this->assertMpttValues($expected, $this->table);
```

Comment:
Do we need to assert that the orm delete was used?

Reply:
Sorry, what do you mean exactly?
I just left the Mptt check in there so we know which tree structure is left after the delete.

Reply:
I adjusted the test to make it more clear, that the ORM delete functionality was used